### PR TITLE
[#1079] Migration

### PR DESCRIPTION
**Pull request by mcdoker18**

### CODE REVIEW

File: containers/terraform/Dockerfile [hunk 1]
```diff
@@ -42,17 +42,10 @@ RUN curl -sL https://packages.microsoft.com/keys/microsoft.asc | apt-key add - &
     echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ $(grep VERSION_CODENAME /etc/os-release | cut -d '=' -f 2) main" | tee /etc/apt/sources.list.d/azure-cli.list && \
     apt-get update -y && apt-get install azure-cli -y && apt-get clean && apt-get autoclean && rm -rf /var/lib/apt/lists/*
 
-# Install AWS CLI and Hiera
-RUN curl -sL https://bootstrap.pypa.io/get-pip.py | python3 && \
-    pip3 install pyyaml==5.1.1 awscli && \
```
Comment (maxkochubey) [inline containers/terraform/Dockerfile]: How can we be without awscli?

File: terraform/modules/k8s/nginx-ingress/aws.tf [hunk 2]
```diff
@@ -4,7 +4,7 @@ locals {
 
 # Data
 data "aws_vpc" "default" {
-  count  = local.aws_resouce_count
+  count = local.aws_resouce_count
```
Comment (maxkochubey) [inline terraform/modules/k8s/nginx-ingress/aws.tf]:
```suggestion
  count = local.aws_resource_count
```

Comment (mcdoker18) [reply terraform/modules/k8s/nginx-ingress/aws.tf]: Fixed

File: containers/terraform/Dockerfile (hunk 1 above)
Comment (mcdoker18) [reply containers/terraform/Dockerfile]: Fixed

**Pull request merged**